public/app/plugins/datasource/grafana-azure-monitor-datasource/components/LogsQueryEditor/QueryField.tsx

```diff
@@ -2,7 +2,62 @@ import { CodeEditor } from '@grafana/ui';
 import React, { useCallback } from 'react';
 import { AzureQueryEditorFieldProps } from '../../types';
 
+import * as Monaco from 'monaco-editor/esm/vs/editor/editor.api';
+
+// @ts-ignore
+import bridgeUrl from 'file-loader!@kusto/monaco-kusto/release/min/bridge.min.js';
+
+// @ts-ignore
+import kustoJsClientUrl from 'file-loader!@kusto/monaco-kusto/release/min/kusto.javascript.client.min.js';
+
+// @ts-ignore
+import newtonsoftJsonUrl from 'file-loader!@kusto/monaco-kusto/release/min/newtonsoft.json.min.js';
+
+// @ts-ignore
+import kustoLanguageBridgeUrl from 'file-loader!@kusto/monaco-kusto/release/min/Kusto.Language.Bridge.min.js';
```

Comment:
[Instructions say](https://github.com/Azure/monaco-kusto/tree/master/samples/react#readme) to put these scripts in your index.html, but that's not feasible for us so I inject them at runtime.

TODO:
 - fix this for typescript
 - clean up the script loading (don't load again if already in the page)